js/entity_browser.new_display.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file entity_browser.entity_reference.js
+ *
+ */
+(function ($, Drupal, drupalSettings) {
+
+  "use strict";
+
+  /**
+   * Registers behaviours related to selected entities.
+   */
+  Drupal.behaviors.entityBrowserNewDisplay = {
+    attach: function (context) {
+      $(context).find('.entity-browser-test-files-form').each(function () {
```

Comment:
This class target specific entity browser in examples module. We need to target generally. Use ".entity-browser-form" instead.